imports/ui/accounts/accounts-templates.html

```diff
@@ -36,7 +36,7 @@
 
 <template name="override-atTitle">
   <h1 class="title-auth">{{title}}</h1>
-  <p class="subtitle-auth">Signing in allows you to have private lists</p>
+  <p class="subtitle-auth">{{_ 'Signing in allows you to have private lists'}}</p>
```

Comment:
I'm not expert on i18n but I think people often do this as a first pass of translating the app rather than figuring out a key for every single string in the app.  You can use "Signing in ... " as a key in the i18n file of course. And you can change that key later if the string changes.